libexec/cli/exec.exec

```diff
@@ -83,5 +83,10 @@ SINGULARITY_IMAGE="${1:-}"
 export SINGULARITY_IMAGE
 shift
 
+if [ -d "$SINGULARITY_IMAGE" ]; then
+  export SINGULARITY_USERNS=1
```

Comment:
Does a use-case exist where someone would want to run in a directory and still use the standard sexec?